### ISSUE TITLE
Can we keep the code clean please?

### ISSUE THREAD

**Issue by ChiefTwoPencils** [DROCCK/oh-behive#22]
I noticed when I was looking at the drop create that was done last Sat that there's scripts scattered about the page. Can we work to keep those in a central area (preferably at the top) and then move them to either the "base.css" if appropriate or create a new file like "map.css" and then link it in the pages where you need it. That can become a maintenance nightmare.

**Comment by dmksystems** [comment]
yeah I am sorry about that was just trying to get it work I will take a look at cleaning it up on thursday after I am done with my midterm.
